configure.ac

```diff
@@ -1586,10 +1586,11 @@ dnl ##########################################################################
 dnl ------------------
 dnl check C-Ares library
 dnl ------------------
+c_ares_found=false
 PKG_CHECK_MODULES([CARES], [libcares], [
   c_ares_found=true
 ],[
-  c_ares_found=false
+  AC_MSG_ERROR([libc-ares is required for building FRR])
```

Comment:
Since when do we actually **need** `libc-ares`?  It's only needed for `nhrpd` and the bgp BMP module (?)